harness/scripts/upload_to_mlflow.py

```
@@ -151,8 +156,18 @@ def upload_directory(
     print()
 
-    experiment_id = get_or_create_experiment(client, experiment_name)
-    run_id = create_new_run(client, experiment_id, run_name=run_name)
+    # Use existing run or create a new one
+    is_existing_run = run_id is not None
+    if is_existing_run:
+        print(f"Attaching artifacts to existing run: {run_id}")
+        # Validate that the run exists
+        try:
+            client.get_run(run_id)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)